arviz/plots/distcomparisonplot.py

```diff
@@ -134,6 +135,19 @@ def plot_dist_comparison(
         for data, var in zip(datasets, var_names)
     ]
 
+    total_plots = sum(
+        1 for _ in xarray_sel_iter(datasets[0], var_names=var_names[0], combined=True)
+    )
```

Comment:
I forgot to multiply in the sample code. This counts the number of variables which needs to be multiplied by `(len(groups) + 1)` to be equal to the total number of subplots that would be needed if we weren't truncating the lists of plotters (also read the next comment about the warning message).